doc/getting-started/Toolchain-installation-on-macOS.md

```diff
@@ -20,10 +20,17 @@ $ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/inst
 Using homebrew, install a bunch of helper tools, such as git, srecord, doxygen (to build the API documentation), mosquitto (to test Contiki-NG's MQTT functionality), tuntap (for tunslip and to run native examples with networking), rlwrap (for shell history), python (python 3 for running scripts and pip).
 
 ```bash
-$ brew install git git-lfs srecord doxygen uncrustify ant mosquitto wget libmagic rlwrap python
+$ brew install git git-lfs srecord doxygen uncrustify ant mosquitto wget libmagic rlwrap python make gawk
 $ brew tap caskroom/cask
 $ brew cask install tuntap
 ```
+
+Observe that you may need to update your PATH environment variable to use GNU Make when running the "make" command. This can be set in your shell initialization file (e.g., ~/.zshrc) as follows.
```

Comment:
I think the "regular" make is also make, it's just version 3.x. Perhaps use "GNU Make 4" in this text to make it explicit that it's a different version from homebrew?

Reply:
Yes, I added the version now.